_posts/2021/2021-03-22-be-specific-when-defining-routes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Be specific when defining your routes"
+description: "Reduce accidental paths and be intentional"
+layout: article
+category: ruby
+image:
+  base: "2021/be-specific-when-defining-your-routes"
+  alt: "Purple-lit urban road junction"
+  source: "https://unsplash.com/photos/_QoAuZGAoPY"
+  credit: "Denys Nevozhai"
+---
+
+Many controllers don't need the full set of restful routes. We often have a resource that can only be created and deleted or where there's only a show action.
```

Comment:
```suggestion
Many controllers don't need the full set of restful routes to be generated. We often have resources in our codebase that can only be created or deleted, or where there's only a show action.
```

Reply:
My suggestion isn't quite there yet. But you're essentially trying to say "if we only need for it to be created or deleted, why also have all the other routes", right?